public/locales/en/settings.json

```diff
@@ -61,6 +61,7 @@
     "placeholder": "Enter Monero address",
     "title": "Monero address"
   },
+  "monero-seed-words": "Monero Seed Words",
```

Comment:
`npm run translate settings monero-seed-words="Monero Seed Words"` to generate for all languages